tests/test_sourmash.py

```diff
@@ -5802,8 +5803,28 @@ def test_search_ani_containment(c):
         assert row['ani'] == "0.9868883523107224"
 
 
-@utils.in_tempdir
-def test_search_ani_containment_fail(c):
+def test_search_ani_containment_asymmetry(runtmp):
+    # test contained_by asymmetries, viz #2215
+    query_sig = utils.get_test_data('47.fa.sig')
+    merged_sig = utils.get_test_data('47-63-merge.sig')
+
+    runtmp.sourmash('search', query_sig, merged_sig, '-o',
+                    'query-in-merged.csv', '--containment')
+    runtmp.sourmash('search', merged_sig, query_sig, '-o',
+                    'merged-in-query.csv', '--containment')
+
+    with sourmash_args.FileInputCSV(runtmp.output('query-in-merged.csv')) as r:
+        query_in_merged = list(r)[0]
+
+    with sourmash_args.FileInputCSV(runtmp.output('merged-in-query.csv')) as r:
+        merged_in_query = list(r)[0]
+
+    assert query_in_merged['ani'] == '1.0'
+    assert merged_in_query['ani'] == '0.9865155060423993'
+
```

Comment:
this is the one I thought would be wrong ... I'm convinced.

Reply:
🎉